specification/appplatform/resource-manager/Microsoft.AppPlatform/preview/2019-05-01-preview/appplatform.json

```diff
@@ -2175,6 +2175,10 @@
           "$ref": "#/definitions/ClusterResourceProperties",
           "description": "Properties of the Service resource",
           "x-ms-client-flatten": false
+        },
+        "sku": {
```

Comment:
>"sku" [](start = 8, length = 5)

you also need to implement the /skus API so the customer can discover the available SKUS.  More information here:  [https://armwiki.azurewebsites.net/api_contracts/SKUUpdateContract%E2%80%8B%E2%80%8B.html?q=sku](https://armwiki.azurewebsites.net/api_contracts/SKUUpdateContract%E2%80%8B%E2%80%8B.html?q=sku)

Reply:
fixed